certbot-dns-linode/setup.py

```diff
@@ -0,0 +1,65 @@
+import sys
+
+from setuptools import setup
+from setuptools import find_packages
+
+version = '0.24.0.dev0'
```

Comment:
This should probably be `0.25.0.dev0` now. (Sorry for the slow turn-around time on this re-review.)